CMakeLists.txt

```diff
@@ -116,7 +116,9 @@ option(ENABLE_OPENWSMAN "Enables the Openwsman extensions." OFF)
 option(ENABLE_AZURE "Enables Azure support." ON)
 option(ENABLE_ENCRYPT_CONFIG "Enables build of encrypt-config binary." ON)
 option(ENABLE_SPLUNK "Enable Splunk support" ON)
+option(ENABLE_ELASTICSEARCH "Enable Elasticsearch support" OFF)
 option(ENABLE_GCP "Enable Google Cloud support" ON)
+option(ENABLE_PROCFS "Enable ProcFs support" OFF)
```

Comment:
This is already added on line 127

Reply:
nice catch, no idea how this happened https://github.com/apache/nifi-minifi-cpp/pull/1349/commits/500deb5ecf704d166196ef141ec2ce5791246bfc

Reply:
It used to be this way on main (didn't check the origin, probably a messed up merge), and I've recently removed the duplicate in #1341. Maybe you saw the diff only and added it back or merged from an older version?